### PR TITLE
update cassandra dependencies

### PR DESCRIPTION
**Pull request by patriknw**
* cassandra-driver-core 3.1.0
* cassandra-all 3.7 (for testing)

### CODE REVIEW

**Comment by patriknw** [comment]
Also marked the statements as idempotent, see #124

File: src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala [hunk 1]
```diff
@@ -76,19 +76,19 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
...
+  def preparedWriteMessage = session.prepare(writeMessage).map(_.setIdempotent(true))
+  def preparedDeleteMessages = session.prepare(deleteMessages).map(_.setIdempotent(true))
   def preparedSelectMessages = session.prepare(selectMessages)
-    .map(_.setConsistencyLevel(readConsistency))
+    .map(_.setConsistencyLevel(readConsistency).setIdempotent(true))
   def preparedCheckInUse = session.prepare(selectInUse)
-    .map(_.setConsistencyLevel(readConsistency))
-  def preparedWriteInUse = session.prepare(writeInUse)
+    .map(_.setConsistencyLevel(readConsistency).setIdempotent(true))
+  def preparedWriteInUse = session.prepare(writeInUse).map(_.setIdempotent(true))
```
Comment (ktoso) [inline src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala]: Interesting,

**Comment by ktoso** [comment]
LGTM

**Comment by patriknw** [comment]
Travis seems to be broken at the moment

**Comment by patriknw** [comment]
To bad that I can't get this validated on travis, but I would like to cut the release so merging. Tests are of course passing on local.

**Pull request merged**